.github/CODEOWNERS

```diff
@@ -106,6 +106,7 @@
 /core/templates/pages/privacy-page/privacy-page.mainpage.html @seanlip
 /core/templates/pages/terms-page/ @DubeySandeep
 /core/templates/pages/terms-page/terms-page.mainpage.html @seanlip
```

Comment:
Do I need to be on this one? Probably not because it hasn't got any of the actual content?

Reply:
Also just to check -- why is this different for the privacy vs the terms page? I.e. why is one changing but not the other?

Reply:
> Do I need to be on this one? Probably not because it hasn't got any of the actual content?

ok, removed.

Reply:
> is this different for the privacy vs the terms page? I.e. why is one changing but not the other?

because `terms` page had jquery code. (Refer https://github.com/oppia/oppia/pull/10141#discussion_r464764232)

Reply:
Ah I see. OK, thanks!